tests/kernel/interrupt/src/interrupt_offload.c

```diff
@@ -17,7 +17,11 @@ static K_THREAD_STACK_DEFINE(wq_stack, STACK_SIZE);
 static K_THREAD_STACK_DEFINE(tstack, STACK_SIZE);
 static struct k_thread tdata;
 
+static struct k_sem sync_sem;
 static struct k_sem end_sem;
+static int wait_for_end;
```

Comment:
Boolean valued variables should use `bool` type.

Reply:
got it fixed! thanks!